intervention_compassion/models/compassion_intervention.py

```diff
@@ -305,6 +305,29 @@ def unlink(self):
     #                             PUBLIC METHODS                             #
     ##########################################################################
 
+    @api.model
+    def create_intervention(self, commkit_data):
+        intervention_mapping = mapping.new_onramp_mapping(
+            self._name,
+            self.env,
+            'intervention_mapping')
+
+        interventionDetailsRequest = commkit_data[
+            'GPInitiatedInterventionHoldNotification']
+
+        vals = intervention_mapping.get_vals_from_connect(
+            interventionDetailsRequest)
+
+        if not 'service_level' in vals:
```

Comment:
intervention_compassion/models/compassion_intervention.py:321:12: E713 test for membership should be 'not in'